post-processor/vagrant/parallels.go

```diff
@@ -41,7 +41,7 @@ func (p *ParallelsProvider) Process(ui packersdk.Ui, artifact packersdk.Artifact
 		}
 
 		tmpPath := filepath.ToSlash(path)
-		pathRe := regexp.MustCompile(`^(.+?)([^/]+\.(pvm|macvm)/.+?)$`)
+		pathRe := regexp.MustCompile(`[^/]+\.(pvm|macvm)/.+$`)
```

Comment:
is `pvmPath = filepath.FromSlash(matches[2])` still correct since there is only one capture would this be at index 1? 

I think it would be great to add a test for this. It wasn't clear to me what the actual file paths can be.

Reply:
That is a good question, I tested this with the updated test that I added to #107, and it did not fail fwiw, but I'll check what happens here. I would think there's two matches only indeed, the global one and the pvm/macvm.
I'll investigate, thanks for the callout

Reply:
Alright nevermind, I guess I didn't do my test right and it does crash now, I'll fix that now

Reply:
Rerolled, since we don't capture the whole path now, but instead only the `.+.pvm|macvm/...`, we don't need to get a submatch and instead we can use `FindString` to return the whole path, excluding the parent directories.

Reply:
Note: I based this PR on #107 for tests to run, we can rebase it on main before merging

Reply:
Thanks for making the update. I rebased onto the latest main and will merge once all goes green.